siatest/renter/renter_test.go

```diff
@@ -1203,3 +1204,77 @@ func TestRedundancyReporting(t *testing.T) {
 		t.Fatal("Redundancy is not increasing")
 	}
 }
+
+// testClearDownloadHistory makes sure that the download history is
+// properly cleared when called through the API
+func testClearDownloadHistory(t *testing.T, tg *siatest.TestGroup) {
+	// Grab the first of the group's renters
+	r := tg.Renters()[0]
+
+	rdg, err := r.RenterDownloadsGet()
+	if err != nil {
+		t.Fatal("Could not get download history:", err)
+	}
+	numDownloads := len(rdg.Downloads)
+	if numDownloads == 0 {
+		// Upload and download files to show build download history
+		var remoteFiles []*siatest.RemoteFile
+		for i := 0; i < 10; i++ {
+			dataPieces := uint64(1)
+			parityPieces := uint64(1)
+			fileSize := 100 + siatest.Fuzz()
+			_, rf, err := r.UploadNewFileBlocking(fileSize, dataPieces, parityPieces)
```

Comment:
If we decide to keep a list instead of a map we should probably don`t have to upload additional files but instead just download the same files multiple times. For that we might be able to reuse existing uploaded files from previous tests.